src/Routes/index.js

```diff
@@ -13,6 +13,7 @@ import {
   NavBarView,
   EmptyView,
   TokensListView,
+  DetailView,
```

Comment:
Rename to `FileDetailView` so it's more specific and easy to follow as the project grows